contracts/interfaces/ISpender.sol

```diff
@@ -2,6 +2,17 @@
 pragma solidity >=0.7.0;
 
 interface ISpender {
+    // System events
+    event TimeLockActivated(uint256 activatedTimeStamp);
+    // Operator events
+    event SetPendingOperator(address pendingOperator);
```

Comment:
```suggestion
    // Owner events
```

Reply:
Fixed in d7008e9

Reply:
`SetPendingOperator` event is not used anymore?

Reply:
Oh right, fixed in 1aa8b3a.